### PR TITLE
Room offset

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Wait till #12 is merged.

Merge together with: https://github.com/tue-robotics/sergio_bringup/pull/24 and https://github.com/tue-robotics/amigo_bringup/pull/104

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Build will fail, because this one is branched of `SDF` branch

**Comment by MatthijsBurgh** [comment]
@jlunenburg your docstrings looks good. Can you approve the PR?

File: src/navigation_plugin.cpp [hunk 1]
```diff
@@ -155,9 +156,11 @@ void NavigationPlugin::configure(tue::Configuration config)
         config.value("max_z", max_z);
 
         config.value("default_offset", default_offset_);
+        if(!config.value("room_offset", room_offset_, tue::config::OPTIONAL))
```
Comment (jlunenburg) [inline src/navigation_plugin.cpp]: Does this mean the `room_offset` needs to be in the `occupancy_grid_publisher` namespace? It shouldn't be there in any case...

**Review by jlunenburg — DISMISSED** [review]
Think we should merge this for now. Improvements are required, see https://github.com/tue-robotics/ed/issues/83 . No time (yet) to fix this decently before RWC2019

File: src/navigation_plugin.cpp (hunk 1 above)
Comment (MatthijsBurgh) [reply src/navigation_plugin.cpp]: Oh Will fix this before tomorrow evening. Then you can check the changes. re-approve and merge.

Comment (MatthijsBurgh) [reply src/navigation_plugin.cpp]: Done

**Review by jlunenburg — APPROVED** [review]

**Pull request merged**